src/plays/image-gallery-using-unsplash-api/styles.css

```diff
@@ -0,0 +1,113 @@
+/* enter stlyes here */
+* {
```

Comment:
Please do not override the styles this way. Add a unqie class name in front of all the styles in all the CSS files so that the override changes are less.

You can add an additional style name here and use that to prefix all styles in all CSS files.

// src/plays/image-gallery-using-unsplash-api/ImageGalleryUsingUnsplashApi.js
```html
<div className="play-details-body">
```